includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1706,4 +1706,25 @@ public function throw_localized_message( $response, $order ) {
 
 		throw new WC_Stripe_Exception( print_r( $response, true ), $localized_message );
 	}
+
+	/**
+	 * Generates a localized message for an error from a response.
+	 *
+	 * @since 4.3.2
+	 *
+	 * @param stdClass $response The response from the Stripe API.
+	 *
+	 * @return string The localized error message.
+	 */
+	public function get_localized_error_message_from_response( $response ) {
```

Comment:
Here's a bug I found while writing tests! This function should actually be here in the base class instead of `WC_Gateway_Stripe`, because it can be called from within either gateway classes via `throw_localized_message`.

Reply:
Nice find! Yay tests!!!